lib/singlepass-backend/src/emitter_x64.rs

```diff
@@ -562,15 +566,15 @@ impl Emitter for Assembler {
             (Size::S64, Location::Memory(src, disp), Location::GPR(dst)) => {
                 dynasm!(self ; lea Rq(dst as u8), [Rq(src as u8) + disp]);
             }
-            _ => unreachable!(),
+            _ => panic!("LEA {:?} {:?} {:?}", sz, src, dst),
```

Comment:
Can you make the panics a bit more descriptive?

Just appending `singlepass reached `:

```suggestion
            _ => panic!("singlepass reached LEA {:?} {:?} {:?}", sz, src, dst),
```

Reply:
Done.